wrench/src/wrench.rs

```diff
@@ -218,6 +193,38 @@ impl Wrench {
         wrench
     }
 
+    pub fn layout_simple_ascii(&self, font_key: FontKey, text: &str, size: Au) -> (Vec<u32>, Vec<f32>) {
+        // Map the string codepoints to glyph indices in this font.
+        // Just drop any glyph that isn't present in this font.
+        let indices: Vec<u32> = self.api
+                                    .get_glyph_indices(font_key, text)
+                                    .iter()
+                                    .filter_map(|idx| *idx)
+                                    .collect();
+
+        // Retrieve the metrics for each glyph.
+        let mut keys = Vec::new();
+        for glyph_index in &indices {
+            keys.push(GlyphKey::new(font_key,
+                                    size,
+                                    ColorF::new(0.0, 0.0, 0.0, 1.0),
+                                    *glyph_index,
+                                    LayerPoint::zero(),
+                                    FontRenderMode::Alpha));
+        }
+        let metrics = self.api.get_glyph_dimensions(keys);
+
+        // Extract the advances from the metrics. The get_glyph_dimensions API
+        // has a limitation that it can't currently get dimensions for non-renderable
+        // glyphs (e.g. spaces), so just use a rough estimate in that case.
+        let space_advance = size.to_f32_px() / 3.0;
```

Comment:
what is the point of making this rough estimate if the only usage of the function (AFAIS) is debug info output?